learn/on-premise/observability/metrics-collection/alerts.md

```diff
@@ -85,3 +85,13 @@ Following are the list of alerts included in WaveMaker
 :::note
 The Alerts listed above keep changing as new things are discovered and might vary over time
 :::
+
+### Configuring Alerts
```

Comment:
Heading : Configure Notifications for Alerts.
https://prometheus.io/docs/alerting/latest/notification_examples/

We are allowing our customers are configure notifications to various channel.  EMail is once channel.
there are other channel as listed in thier website.  So you understand from above link and write it down.